src/test/java/com/truelayer/java/acceptance/PayoutsAcceptanceTests.java

```diff
@@ -0,0 +1,62 @@
+package com.truelayer.java.acceptance;
+
+import static com.truelayer.java.TestUtils.assertNotError;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import com.truelayer.java.entities.CurrencyCode;
+import com.truelayer.java.http.entities.ApiResponse;
+import com.truelayer.java.merchantaccounts.entities.MerchantAccount;
+import com.truelayer.java.payouts.entities.CreatePayoutRequest;
+import com.truelayer.java.payouts.entities.CreatePayoutResponse;
+import com.truelayer.java.payouts.entities.Payout;
+import com.truelayer.java.payouts.entities.accountidentifier.AccountIdentifier;
+import com.truelayer.java.payouts.entities.beneficiary.Beneficiary;
+import java.util.Collections;
+import lombok.SneakyThrows;
+import org.apache.commons.lang3.RandomUtils;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Tag;
+import org.junit.jupiter.api.Test;
+
+@Tag("acceptance")
+public class PayoutsAcceptanceTests extends AcceptanceTests {
+
+    @Test
+    @DisplayName("It should create a payout and get the details")
+    @SneakyThrows
+    public void shouldCreateAPayoutAndGetPayoutsDetails() {
+        // find a merchant to execute the payout from
+        MerchantAccount merchantAccount = getMerchantAccount(CurrencyCode.GBP);
+
+        // create the payout
+        CreatePayoutRequest createPayoutRequest = CreatePayoutRequest.builder()
+                .merchantAccountId(merchantAccount.getId())
+                .amountInMinor(RandomUtils.nextInt(10, 100))
+                .currency(CurrencyCode.GBP)
+                .beneficiary(Beneficiary.externalAccount()
+                        .accountIdentifier(AccountIdentifier.sortCodeAccountNumber()
+                                .accountNumber("00009650")
+                                .sortCode("040668")
+                                .build())
+                        .reference("Java SDK payout test")
+                        .accountHolderName("LucaB merchant")
```

Comment:
Using Luca Baggi Sandbox's merchant account as destination for the payout. We can change this in the future if we decide to create a specific merchant for this use case